src/Domain/Creators/BinaryCompactObject.cpp

```diff
@@ -254,6 +260,8 @@ BinaryCompactObject::BinaryCompactObject(
               << ". Set it to 'Auto' so a reasonable value is chosen "
                  "automatically.");
     }
+  } else if (frustum_sphericity == 1.) {
+    radius_enveloping_sphere_ = radius_enveloping_cube_;
```

Comment:
maybe for clarity add that the enveloping sphere is not the outermost sphere as there is still another layer

Reply:
There's a comment above this if-else block that says "Compute the inner radius of the outer spherical shell". Don't you think that's sufficient?

Reply:
Sorry, I missed that! What's there is sufficient :slightly_smiling_face: